README.md

```diff
@@ -4,7 +4,7 @@ This repository contains source templates and a Python utility to generate HTML
 
 Join the Ansible community and help us build a better user experience for the docsite.
 
-We talk about Ansible documentation on Matrix at [#docs:ansible.im](https://matrix.to/#/#docs:ansible.im) and on the [Ansible Community Forum](https://forum.ansible.com/tag/documentation).
-You're welcome to join us, chat about the Ansible docs, and share your thoughts.
+We talk about Ansible documentation on the [Ansible Community Forum](https://forum.ansible.com/tag/documentation).
```

Comment:
```suggestion
We talk about Ansible documentation on the [Ansible Forum](https://docs.ansible.com/ansible/devel/community/communication.html#forum).
```
I would suggest always referring to the communication guide in such cases as the main and only source of truth + it provides some important context
And it's not community anymore;)

Reply:
Two comments:
1 - that docs page doesn't include the specific working group details, so we need to point to the forum tag directly.
2. The more forum links we use, the higher our forum results show up on Google and other searches.